starter-code/src/assessment1.js

```diff
@@ -1,27 +1,48 @@
 // Write a function that returns the product of 2 numbers
-function product(x, y) {
+function product(x, y) { 
+  return x * y;
   //
-}
+};
 
 // Write a function that returns whether a given number is even
 function isEven(num) {
-  //
+if (num%2===0){
+  return true;
+} else {
+  return false;
 }
+};
 
 // Return the largest of 2 numbers
 function maxOfTwoNumbers(a, b) {
-  //
-}
+  if(a>b){
+    return a;
+  }else if (b>a){
+    return b;
+  }
+};
 
 // Return the largest of 3 numbers
 function maxOfThreeNumbers(a, b, c) {
-  //
-}
+ if (a>b && a>c){
+   return a;
+ } else if (b>a && b>c){
+   return b; 
+ } else {
+   return c;
+ }
+};
 
 // Calculate the sum of an array of numbers
 function sumArray(numbers) {
-  //
-}
+    let addition = 0;
+  
+  for(let i=0; i < numbers.length; i++) {
+  return addition = addition + numbers[i];
```

Comment:
return will end the for loop. just do the addition and return at the end of the function